src/main/java/com/project/store/dto/category/CategoryDto.java

```diff
@@ -12,6 +12,7 @@ public class ProductCategoryDto {
     @NotEmpty(message = "product.category.type.cannot.be.empty")
     private String type;
```

Comment:
Category type is redundant. 
The initial idea was taken from eMAG where one has: TV, Audio-Video & Foto -> TV & Accesories -> TV
This would overcomplicate matters.
It will be removed

Reply:
Name will also be needed.
Regarding category, if you want to make something like this in the future, type will not be enough, you can think of a tree structure, with parent categories.

Reply:
From the initial commit the name field was accidentally removed. Now is back again in the CategoryDto